server/models/message_model.py

```diff
@@ -0,0 +1,29 @@
+from extensions import db
+
+
+class MessageModel(db.Model):
+    __tablename__ = "messages"
+
+    id = db.Column(db.Integer, primary_key=True)
+    review_id = db.Column(db.Integer, nullable=False)
```

Comment:
Notice you can actually defines relationships in SQLAlchemy 
https://docs.sqlalchemy.org/en/13/orm/basic_relationships.html#one-to-many

You should specific if it is a ForeignKey because then it will be more efficient to get all messages for a review, and  if you use the relationship examples above, you get a lot of methods for free